packages/ketcher-core/src/application/editor/Editor.ts

```diff
@@ -51,6 +49,20 @@ export class CoreEditor {
     this.events.selectMonomer.add((monomer) => this.onSelectMonomer(monomer));
     this.events.selectPreset.add((preset) => this.onSelectRNAPreset(preset));
     this.events.selectTool.add((tool) => this.onSelectTool(tool));
+    const renderersEvents: ToolEventHandlerName[] = [
```

Comment:
Let's move it to separate file. Most likely, we will have a lot of different events.